client/layout/index.js

```diff
@@ -222,15 +222,19 @@ export const PageLayout = compose(
 		: identity
 )( _PageLayout );
 
-export class EmbedLayout extends Component {
-	render() {
-		return (
-			<Layout
-				page={ {
-					breadcrumbs: getSetting( 'embedBreadcrumbs', [] ),
-				} }
-				isEmbedded
-			/>
-		);
-	}
-}
+const _EmbedLayout = () => (
+	<Layout
+		page={ {
+			breadcrumbs: getSetting( 'embedBreadcrumbs', [] ),
+		} }
+		isEmbedded
+	/>
+);
+
+export const EmbedLayout = compose(
```

Comment:
We can probably use `window.wcAdminFeatures` to check if the feature is enabled here.

Reply:
I think it was a mistake to not preload options for embedded pages. See https://github.com/woocommerce/woocommerce-admin/pull/5161 for a ramification of that decision. I think it could happen easily again because its easy to overlook an option being pre loaded on the app, but not in the embedded pages. So, I think pre-loading should happen regardless of Navigation or not.